src/main/java/pl/touk/sputnik/processor/spotbugs/SpotBugsProcessor.java

```diff
@@ -121,4 +130,23 @@ private String getExcludeFilterFilename() {
         log.info("Using SpotBugs exclude filter file {}", excludeFilterFilename);
         return excludeFilterFilename;
     }
+
+    private void loadAllSpotbugsPlugins() {
+        String pluginsLocation = config.getProperty(GeneralOption.SPOTBUGS_PLUGINS_LOCATION);
+        if (pluginsLocation != null) {
+            File[] pluginsList = new File(pluginsLocation).listFiles();
+            for (File plugin : Objects.requireNonNull(pluginsList)) {
```

Comment:
What if location is present but there are no files? `Objects.requireNonNull` would throw exception?

Reply:
Thanks for pointing it out, when I've done modification in the way plugins will be loaded, I get rid of that part.